src/MeshCat.jl

```diff
@@ -18,7 +18,7 @@ using Base.Filesystem: rm
 using BinDeps: download_cmd, unpack_cmd
 using UUIDs: UUID, uuid1
 using LinearAlgebra: UniformScaling, norm
-using Sockets: listen
+using Sockets#: listen
```

Comment:
Why?

Reply:
I haven't had time to find the include for `ip"127.0.0.1"` from Sockets